src/Microsoft.Management.Infrastructure.CimCmdlets/AssemblyInfo.cs

```diff
@@ -1,5 +1,5 @@
 /*============================================================================
- * Copyright (C) Microsoft Corporation, All rights reserved.
+ * Copyright (c) Microsoft Corporation, All rights reserved.
```

Comment:
Please replace "," with dot.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

We have many such patterns below.

Reply:
I'll fix all the occurrences